config/spryk/spryks/Spryker/Yves/Plugin/AddSubFormPlugin.yml

```diff
@@ -0,0 +1,34 @@
+spryk: template
+description: "Adds a new *module* SubFormPlugin class."
+mode: both
+
+arguments:
+    module:
+        inherit: true
+
+    organization:
+        inherit: true
+        default: SprykerEco
+
+    className:
+        value: "{{ paymentMethod }}SubFormPlugin"
```

Comment:
this spryk is related payment module, but its says "Adds a new *module* SubFormPlugin class." which means - for every module.

Maybe you can make variable name more generic. Or move this spryk into additional folder `/Yves/Plugin/Payment`

Reply:
done